core/core.go

```diff
@@ -89,18 +89,18 @@ type Localization struct {
 }
 
 type Extension struct {
-	Assets           map[string]Asset `json:"assets" yaml:"-"`
-	CanAccessNetwork bool             `json:"canAccessNetwork" yaml:"can_access_network,omitempty"`
```

Comment:
I'm unsure about whether we should be using `omitempty` for json serialization. I'll let @dmhenry overrule me on this, but perhaps we want to have a `"capablities"` array always appear in the `/extensions` endpoint response? Might be better we be explicit that we have an empty list of capabilities rather than need to worry about null handling on the checkout side.

Also, we appear to follow this same approach in the other two array fields we serialize to json: `ExtensionPoints` and `Metafields`.

Reply:
Hmm, I think we shouldn't accept an array of string but more more strict in the expected shape. Can it be a struct?
```go
type Capabilities struct {
	CanAccessNetwork bool `json:"canAccessNetwork" yaml:"can_access_network,omitempty"`
}

type Extension struct {
	Capabilities    Capabilities     `json:"capabilities" yaml:"capabilities,omitempty"`
}

```